setup.cfg

```diff
@@ -17,3 +17,13 @@ parentdir_prefix = geopandas-
 markers =
     web: tests that need network connectivity
 xfail_strict = True
+
+[flake8]
+# Black enforces 88 characters in code,
+# allow slightly longer lines in comments with URLs
+max_line_length = 99
```

Comment:
I would prefer to keep this at 88 (eg to restrict docstrings), and add a few `# noqa` 's where needed

Reply:
Sounds good. I can update tonight.

Reply:
Fixed